eng/test.yaml

```diff
@@ -28,6 +28,47 @@ steps:
     NUGET_PACKAGES: $(Build.SourcesDirectory)/.packages
   condition: succeededOrFailed()
 
+- powershell: |
+    # Prepare an empty VMR
+    $vmrDir = "$(Agent.TempDirectory)\vmr"
+    New-Item -Path "$vmrDir\src" -ItemType directory
+    Copy-Item -Path "$(Build.SourcesDirectory)\src\Microsoft.DotNet.Darc\tests\Microsoft.DotNet.Darc.Tests\inputs\source-mappings.json" -Destination "$vmrDir\src"
+    git config --global user.email "dotnet-maestro[bot]@users.noreply.github.com"
+    git config --global user.name "dotnet-maestro[bot]"
+    git -C "$vmrDir" init --initial-branch main
+    git -C "$vmrDir" add -A
+    git -C "$vmrDir" commit -m "Initial commit"
+
+    # Get SHAs of last few commits to sync between them
+    $response = Invoke-RestMethod -Uri "https://api.github.com/repos/dotnet/command-line-api/commits?per_page=5"
+    $firstSha = $response[0].sha
+    $secondSha = $response[4].sha
```

Comment:
Trivial comment - you should be able to use -1 as an array index in powershell, so as not to break the command if someone changes the query or for whatever reason the query returns less then 5 elements